test/KeyBindingsTest.cs

```diff
@@ -47,5 +47,21 @@ public void ShowKeyBindings()
             TestSetup(KeyMode.Vi);
             Test("", Keys(Enumerable.Repeat(_.Ctrl_Alt_Question, 3)));
         }
+
+        [SkippableFact]
+        public void ShiftBackspace()
+        {
+            TestSetup(KeyMode.Cmd);
+            Test("aa", Keys("aaa", _.Shift_Backspace));
+        }
+
+        [SkippableFact]
+        public void ShiftEscape()
+        {
+            TestSetup(KeyMode.Cmd);
+
+            System.Threading.Thread.Sleep(10000);
```

Comment:
Remove or comment why this is necessary.

Reply:
It was for diagnosis :) Already removed.